meson.build

```diff
@@ -0,0 +1,50 @@
+project('parson', 'c',
+    version : '1.4.0',
+    license : 'MIT',
+    meson_version : '>=0.54.0',
```

Comment:
Note that the most recent feature of Meson you are using is from 0.46.0, so you can relax this minimum.

Reply:
Got it. I think I used the default meson included with Debian stable at the time. Will do!

Reply:
Here is a secret trick you can use: temporarily edit that down to something like 0.37.0 and then re-run Meson, and Meson will print a list of all the features you are using that are too new for Meson 0.37.0 (this is around when we first started tracking this information), as well as the version of Meson that feature was added in.

Then simply grab the highest version listed and use that as the minimum :D and Meson will also warn you if you later on introduce too new features. At that time you may decide to bump the minimum version, or do without the feature, depending on which versions you find a need to support.

Reply:
That's really helpful. I thought choosing meson was arbitrary based on what system you want to target.

Reply:
It can be if you want. Personally I prefer to be careful here -- just because you're not interested in supporting anything older than debian stable, doesn't mean that if you coincidentally haven't broken support, you should forbid it anyway.

I like to double check and leave open the option that someone on an older system might want to -- and be able to -- use it anyway, at least up until I decide some new Meson feature is really tempting and I want to bump the minimum version in order to use it. At that point, I would say -- don't be afraid to use new features.

Reply:
Any reason for reverting the change to 0.46.0?

Reply:
I'm not using any features that are only in 0.54 or newer.

Reply:
Yeah but aa @eli-schwartz pointed out it should (and does) work with >=0.46.0.

Reply:
My bad. I was messing around with making the pull request one commit and I really messed it up. It should now be fine.